app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,133 @@
+
+## Service Catalog terminology
+
+| Term | Definition |
+| ---- | ---------- |
+| Integration | These are applications, either {{site.konnect_short_name}}-internal or external, that act as sources from which you can ingest resources. For example, GitHub. |
+| Resource | An umbrella term that denotes entities ingested by Service Catalog from enabled integrations. A resource can range from an infrastructural component (like Gateway services, mesh services, databases, and caches) to an external application or tool (like code repositories, CI/CD infrastructure, and on-call systems) to a piece of documentation (like API specs). Resources can be mapped to one or more Services. |
+| Service Catalog service | A unit of software that is typically owned by a single team, exposes one or more APIs, and may be dependent on other Service Catalog services (as either upstream or downstream). A Service Catalog service can be thought of as a collection of one or more resources. |
+| Service Catalog | A comprehensive catalog of all resources and Service Catalog services running in your organization. |
+
+## FAQs
+
+<details><summary>How do Service Catalog services map to API products? What is the relationship there?</summary>
+
+{% capture service_mapping %}
+Service Catalog services do not directly map to API products. Rather, a Gateway service can be mapped to a Service Catalog service and you can then map the Gateway service to an API product version in Service Catalog.
+{% endcapture %}
+
+{{ service_mapping | markdownify }}
+
+</details>
+
+<details><summary>How do I view the health and status of my services in the Service Catalog?</summary>
+
+{% capture service_health %}
+Navigate to **Resources** in the Service Catalog and click on the Service Catalog service you want to view the health and status of.
+{% endcapture %}
+
+{{ service_health | markdownify }}
+
+</details>
+
+<details><summary>How can I identify and reuse existing resources?</summary>
+
+{% capture reuse_resources %}
+Service Catalog will pull in any resources that match your specified criteria when you enable integrations. 
+{% endcapture %}
+
+{{ reuse_resources | markdownify }}
+
+</details>
+
+<details><summary>What should I do if I notice discrepancies in the data shown in the Service Catalog?</summary>
+
+{% capture discrepancies %}
+Check the Service Catalog integration settings and data sources for any issues. Ensure that all connected tools are properly configured and that data synchronization is functioning correctly.
+{% endcapture %}
+
+{{ discrepancies | markdownify }}
+
+</details>
+
+<details><summary>Can I control who has access to specific Service Catalog service or data?</summary>
+
+{% capture service_access %}
+Yes, you can configure access controls and manage permissions to Service Catalog by configuring [teams](/konnect/org-management/teams-and-roles/manage/) and [roles](/konnect/org-management/teams-and-roles/roles-reference/).
```

Comment:
Does this page need to be updated with Service Catalog related roles? /konnect/org-management/teams-and-roles/roles-reference/

Reply:
NVM, just saw `https://github.com/Kong/docs.konghq.com/pull/7849`!